content/release-notes/release-notes.md

```diff
@@ -17,20 +17,18 @@ For more information see [PI Adapter for OPC UA overview](xref:PIAdapterForOPCUA
 
 ### Fixes
 
-The following issues reported from version 1.1 are fixed in this release.
+The following issues reported from version 1.2 are fixed in this release.
```

Comment:
I wouldn't avoid putting a single version number here because it impacted multiple versions.

I'd just say "The following issues are fixed"